src/types/mod.rs

```diff
@@ -65,6 +65,26 @@ impl IonType {
     }
 }
 
+/// Returns the number of base-10 digits needed to represent `value`.
+fn num_decimal_digits_in_u64(value: u64) -> u64 {
```

Comment:
🗺️ As mentioned earlier, this was moved to the root of the module for common access.